### PR TITLE
Avoid deleting existing volume if there is an attempt to create the s…

### PR DESCRIPTION
**Pull request by humblec**
…ame volume again.

Signed-off-by: hchiramm <hchiramm@redhat.com>

### CODE REVIEW

File: executors/sshexec/volume.go [hunk 1]
```diff
@@ -77,7 +78,12 @@ func (s *SshExecutor) VolumeCreate(host string,
 	// Execute command
 	_, err := s.RemoteExecutor.RemoteCommandExecute(host, commands, 10)
 	if err != nil {
-		s.VolumeDestroy(host, volume.Name)
+		volexist := "Volume " + volume.Name + " already exists"
+		if strings.Contains(err.Error(), volexist) {
+			fmt.Printf("\n Volume exist, Please recheck the volume create command")
```
Comment (lpabon) [inline executors/sshexec/volume.go]: Cannot use fmt.Printf() This goes to stdout.

**Comment by lpabon** [comment]
This method is really an educated guess and not the correct way to determine if there is another volume with that name.

**Comment by humblec** [comment]
@lpabon, as long as the error message remains same from Gluster , this should work.

**Comment by lpabon** [comment]
That is still not accurate enough.  Instead I commented how to do it here: https://github.com/heketi/heketi/issues/293#issuecomment-220877895 .

**Comment by humblec** [comment]
@lpabon can you please review/merge this patch?

**Comment by lpabon** [comment]
Let's discuss this tomorrow.

**Comment by humblec** [comment]
@lpabon let me know whenever you are ready to discuss this.

**Comment by humblec** [comment]
@lpabon can you review/merge this patch ?

**Comment by lpabon** [comment]
@humblec Let me take a look if the issue is still there.

**Comment by lpabon** [comment]
This fix is not longer necessary for Issue #293, because it was fixed by PR #388.

**Comment by humblec** [comment]
@lpabon looks like #388 came after this fix was proposed. Didnt know about #388 is fixing this issue as well. Is there a way we can track the patches. Otherwise, its really waste of effort and dont want to come to this same situation again. :(

**Pull request closed without merging**